src/internal/components/pluggableVisualizations/geoChart/PluggableGeoPushpinChart.tsx

```diff
@@ -144,15 +144,20 @@ export class PluggableGeoPushpinChart extends PluggableBaseChart {
                 ...options.resultSpec,
                 dimensions: this.getDimensions(mdObject),
             };
-            const enableSortingByTotalGroup = this.featureFlags.enableSortingByTotalGroup as boolean;
-            const sorts: AFM.SortItem[] = createSorts(
-                this.type,
-                dataSource.getAfm(),
-                resultSpecWithDimensions,
-                allProperties,
-                canSortStackTotalValue(mdObject, null, enableSortingByTotalGroup),
-                enableSortingByTotalGroup,
+
+            const hasMeasureSize = mdObject.buckets.some(
+                bucket => bucket.localIdentifier === BucketNames.SIZE,
             );
+            const sorts: AFM.SortItem[] = hasMeasureSize
+                ? createSorts(
+                      this.type,
+                      dataSource.getAfm(),
+                      resultSpecWithDimensions,
+                      allProperties,
+                      false,
+                      false,
```

Comment:
`canSortStackTotalValue` and `enableSortingByTotalGroup` are false by default so we can ommit its values